### DISCUSSION TITLE
[Feature request] | input output device switch

### DISCUSSION THREAD

**Discussion by renegade1337** [LeonMarcel-HD/StreamDeck-TeamSpeak5#2] [Feature Requests / Suggestions]
I would be happy if you could change the microphone in the Teamspeak app at the push of a button so that you can switch from the headset microphone (wireless) to a stand microphone.

**Comment by LeonMarcel-HD** [comment]
_Thanks for the Feedback, I moved it into a Discussion :)_

Sadly that is currently not possible due to the limitations of Remote Apps. I could see this being possible when Audio Profiles come to TeamSpeak 5 and then set a Keybinding for that Profile.

I've already talked to an TeamSpeak Employee about missing features for the Remote Apps and sadly they don't currently have the capacity to work on that.
